src/v/wasm/schema_registry.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "pandaproxy/schema_registry/api.h"
+#include "pandaproxy/schema_registry/types.h"
+
+namespace wasm {
+class schema_registry {
```

Comment:
i'm just generally a bit confused about why there is a schema registry implementation here..